Commands/general/apply.ts

```diff
@@ -1,11 +1,17 @@
-import { ChatInputCommandInteraction, SlashCommandBuilder, ActionRowBuilder, ModalBuilder, TextInputBuilder, TextInputStyle } from "discord.js";
+import { ChatInputCommandInteraction, SlashCommandBuilder, ActionRowBuilder, ModalBuilder, TextInputBuilder, TextInputStyle, TextChannel } from "discord.js";
 import { DiscordClient } from "../../Utils/DiscordClient";
 
 export const data = new SlashCommandBuilder()
     .setName('apply')
     .setDescription('Apply for the rw role!')
 
 export async function execute(interaction: ChatInputCommandInteraction, client: DiscordClient) {
+    const channel = client.channels.cache.find((channel: any) => channel.topic === interaction.member?.user.id) as TextChannel;
```

Comment:
Is this cache automatically rebuilt when the bot reboots? Otherwise this could result in an edgecase where a second channel still can be made.

Reply:
I tested it, I logged the cache when the bot emits the ready event and the cache was there.